pipeline/outputs/stackdriver.md

```diff
@@ -55,7 +55,7 @@ If you are using a _Google Cloud Credentials File_, the following configuration
 
 Example configuration file for k8s resource type:
 
-local_resource_id is used by stackdriver output plugin to set the labels field for different k8s resource types. Stackdriver plugin will try to find the local_resource_id field in the log entry. If there is no field logging.googleapis.com/local_resource_id in the log, the plugin will then construct it by using the tag value of the log.
+`local_resource_id` is used by stackdriver output plugin to set the labels field for different k8s resource types. Stackdriver plugin will try to find the local_resource_id field in the log entry. If there is no field logging.googleapis.com/local_resource_id in the log, the plugin will then construct it by using the tag value of the log.
```

Comment:
```suggestion
`local_resource_id` is used by the Stackdriver output plugin to set the labels field for different k8s resource types. Stackdriver plugin will try to find the `local_resource_id` field in the log entry. If there is no field `logging.googleapis.com/local_resource_id` in the log, the plugin will then construct it by using the tag value of the log.
```

Reply:
Fixed in [f2f67cf](https://github.com/fluent/fluent-bit-docs/pull/1585/commits/f2f67cf3cd9d5af702c78466174accaa9e401754)

There are likely a number of little cleanups to do here that I will try and find time to do. Nobody who has worked on these docs from the Google side is a technical writer by trade (nor am I) so a lot of it could be cleaner. Fixing these things ends up being sadly a bit of a side priority, but I will still try and come back and clean stuff up when the opportunity presents itself.